source/shared/net/guides/become-baker.rst

```diff
@@ -300,6 +303,34 @@ Participating in the finalization committee produces rewards on each block that
 is finalized. The rewards are paid to the baker account some time after the
 block is finalized.
 
+Update baker keys
+-----------------
+
+If it is necessary to update your baker keys, you need to first generate new baker keys. To create a fresh set of keys run:
+
+.. code-block:: console
+
+   $concordium-client baker generate-keys <keys-file>.json
+
+You can choose an arbitrary name for the ``keys file``.
+
+Then run the transaction:
+
+.. code-block:: console
+
+   $concordium-client baker set-key <keys-file>.json --sender <account> --out <concordium-data-dir>/baker-credentials.json
+
+``--sender`` is the name or address of the transaction's sender account. The name is the one that's used when you :ref:`import the account<concordium-client-import-accounts-keys>` (assuming that this
```

Comment:
Technically `sender` can also be an address. But maybe mentioning that would detract from the flow here?

Reply:
Yeah, I think so. Let's get it out there and see if we need to clarify it.